### DISCUSSION TITLE
Basic guide

### DISCUSSION THREAD

**Discussion by AkniSteven** [TENK-DAO/tenk#24] [Q&A]
Hi, Guys. I need some help with https://github.com/TENK-DAO/tenk
1) How to setup project and witch requirements does it have? I installed yar, cargo, rustc, but on trying run pretest I received an issue " error[E0463]: can't find crate for core
  |
  = note: the wasm32-unknown-unknown target may not be installed
" So please help me solve this one.
2) there are a lot of commands in the package json, please explain which of them will help me to create and upload nfts collection for my near account and exactly it will be grate to have short guide how to use it step-by-step

**Comment by willemneal** [comment]
1. need to add the target with rustup.  see: https://www.near-sdk.io/
2. Working on it!  `yarn build` is for testnet, `yarn build:mainnet` for mainnet.
